lib/class-wp-rest-widget-types-controller.php

```diff
@@ -421,6 +421,9 @@ public function encode_form_data( $request ) {
 			);
 		}
 
+		// Some third-party widgets rely on wp-admin functions so let's load them before rendering the preview.
```

Comment:
"rendering the preview" is just one of the functionalities being invoked during this request. I find this comment to be confusing and might lead to someone down the road moving the code to the previewer method.

How about something like this?

```suggestion
		// Third-party widgets may rely on wp-admin functions. So let's load them before working with the widget object.
```